src/LRS.js

```diff
@@ -341,13 +378,43 @@ TinCan client library
                 };
             }
 
-            requestCfg = {
-                url: "statements",
-                data: JSON.stringify(versionedStatement),
-                headers: {
-                    "Content-Type": "application/json"
+            if (versionedStatement.hasOwnProperty("attachments") && versionedStatement.attachments !== null) {
+                boundary = this.getBoundary();
+                requestCfg = {
+                    url: "statements",
+                    headers: {
+                        "Content-Type": "application/json"
+                    }
+                };
+                for (a = 0; a < versionedStatement.attachments.length; a += 1) {
+                    if (typeof versionedStatement.attachments[a].content !== "undefined" && versionedStatement.attachments[a].content !== null) {
+                        attachmentContent.push(versionedStatement.attachments[a].content);
+                        delete versionedStatement.attachments[a].content;
```

Comment:
This begs the question of whether the versioned statement should *ever* have the `content` set, since it can't be used in that way. It probably shouldn't, and instead we use the index of the versioned attachments to index into the original statement to check for content. See PHP for an example.